Sources/Card/CardClient.swift

```diff
@@ -27,11 +27,11 @@ public class CardClient {
     ///   - orderID: The ID of the order to be approved
     ///   - card: The card to be charged for this order
     ///   - completion: Completion handler for approveOrder, which contains data of the order if success, or an error if failure
-    public func approveOrder(orderID: String, card: Card, completion: @escaping (Result<CardResult, PayPalSDKError>) -> Void) {
+    public func approveOrder(request: CardRequest, completion: @escaping (Result<CardResult, PayPalSDKError>) -> Void) {
```

Comment:
If we wanted to do that rename to use `with` externally we could just do this here:
```
public func approveOrder(with request: CardRequest, completion: @escaping (Result<CardResult, PayPalSDKError>) -> Void) {
```

Reply:
I'm up for it! I guess one thing I didn't consider was the alignment with Android. It may make sense to circle back once more of the API is done.

The SwiftUI guidelines are super in-depth. I think it makes sense for iOS to take lead on overall API structure and we can take the same blueprint and transform it for Android since named parameters are handled a bit differently in Kotlin (and are nonexistent in Java).